dbms/src/Encryption/MockTiFlashRaftProxyHelper.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+#include <pingcap/pd/Types.h>
+
+#include <string>
+#include <unordered_set>
+
+
+namespace DB
+{
+
+using String = std::string;
+using KeyspaceID = pingcap::pd::KeyspaceID;
+
+struct MockTiFlashRaftProxyHelper
```

Comment:
Why not put it in `MockRaftStoreProxy`? It is in `RaftStoreProxy` rather than `RaftStoreProxyHelper` in your pr https://github.com/tidbcloud/cloud-storage-engine/pull/1167

Reply:
I think the name is not important, I just want to keep it simple, since we only need the two FFI here.

Reply:
Then I suggest we don't name it `MockTiFlashRaftProxyHelper`, give a private name maybe, since others may not use this class for now.
Meanwhile, I think it should be put into `gtest_` or `Debug/`

Reply:
ok, rename to `MockProxyEncryptionFFI`, move to `Debug/`